README.md

```diff
@@ -99,6 +99,27 @@ For dependency details, look at `extras` variable in [`setup.py`](https://github
   - [Upgrade ThaiNER from 1.7](https://github.com/PyThaiNLP/pythainlp/wiki/Upgrade-ThaiNER-from-PyThaiNLP-1.7-to-PyThaiNLP-2.0)
 - Python 2.7 users can use PyThaiNLP 1.6
 
+## Citations
+
+If you use `PyThaiNLP` in your project or publication, please cite the library as follows
+
+```
+Wannaphong Phatthiyaphaibun, Korakot Chaovavanich, Charin Polpanumas, Arthit Suriyawongkul, Chakri Lowphansirikul, & Pattarawat Chormai. (2016, Jun 27). PyThaiNLP: Thai Natural Language Processing in Python. Zenodo. http://doi.org/10.5281/zenodo.3519354
+```
+
+or BibTeX entry:
+
+``` bib
+@misc{Wannaphong,
```

Comment:
This line can be changed to `@misc{pythainlp,`.
Normally the name at the first line will be used as a handle for reference.
So it should be something rather unique. ("Wannaphong" may have other papers or softwares).

Reply:
Done editing.